### DISCUSSION TITLE
Reset ID Counter

### DISCUSSION THREAD

**Discussion by FuckingToasters** [diced/zipline#476] [Ideas]
Hello, it would be nice to have an option which recounts the IDs because currently if you make one folder, delete it and make another one, the id from other other folder is 2 and not 1.

Same happen with uploading images, texts, files, codes.
![image](https://github.com/diced/zipline/assets/94435104/37f58284-918b-44b6-a0c6-ba90a4405c14)

Regards,
FuckingToasters

**Comment by diced** [comment]
i don't think it's possible due to the way autogenerated ids work in postgres. if you want you can look up a method on doing it but zipline won't do it for you. the old id system is being removed in v4 anyway so you won't have this problem when it's out!
